lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -81,6 +81,27 @@ export enum InteractionType {
     Silent = "silent"
 }
 
+/**
+ * Types of interaction currently in progress.
+ * Used in events in wrapper libraries to invoke functions when certain interaction is in progress or all interactions are complete.
+ */
+export enum InteractionStatus {
+    // Initial status before interaction occurs
+    Startup = "startup",
+    // Status set when all login calls occuring
+    Login = "login",
+    // Status set when logout call occuring
```

Comment:
Not sure these individual comments are totally necessary but if you change these from single line comments `//` to block comments `/**/` they'll show up in the typedocs as well.

Reply:
Definitely would like them to show up in typedocs and intellisense.

Reply:
Makes sense. Changed it, along with a few others in the same file.